jkube-kit/enricher/api/src/test/java/org/eclipse/jkube/kit/enricher/handler/CronJobHandlerTest.java

```diff
@@ -97,7 +98,7 @@ void get_withValidConfigs_shouldReturnConfigWithContainers() {
             .extracting(JobSpec::getTemplate)
             .extracting(PodTemplateSpec::getSpec)
             .hasFieldOrPropertyWithValue("restartPolicy", "Never")
-            .extracting(PodSpec::getVolumes).asList().first()
+            .extracting(PodSpec::getVolumes).asInstanceOf(InstanceOfAssertFactories.list(List.class)).first()
```

Comment:
type inside `list(...)` should be type of List enclosed in `extracting`.

In this case it should be `Volume`

Reply:
Thank you for help! I fixed it.